client/src/AppRoot.tsx

```diff
@@ -28,7 +28,7 @@ const AppRoot = ({ storage }: Props) => {
 
   const storageElection = getElection()
   const [electionHash, setElectionHash] = useState(
-    storageElection ? ObjectHash(storageElection) : ''
+    storageElection ? sha256(JSON.stringify(storageElection)) : ''
```

Comment:
you almost certainly want to hash the election on the way in as a string, before you make it into an object. If you make it into an object then stringify it back out, you're not guaranteed to have the same original string.